cluster.yml

```diff
@@ -2,6 +2,15 @@
 - name: Check ansible version
   import_playbook: ansible_version.yml
 
+- name: Add kube-master nodes to kube_control_plane
```

Comment:
I see that this task/play is copied pasted in multiple file, could you factor that in a file and use import_playbook ?
This will help to finish moving away from invalid group name

Reply:
I want to keep here as it and will do cleanup with another PR because this already takes much time.

Reply:
ok for me